checkov/serverless/runner.py

```diff
@@ -111,6 +113,18 @@ def run(
             # Filter out empty files that have not been parsed successfully
             self.definitions = {k: v for k, v in definitions.items() if v}
             self.definitions_raw = {k: v for k, v in definitions_raw.items() if k in definitions.keys()}
+            self.context = build_definitions_context(definitions=self.definitions, definitions_raw=self.definitions_raw)
+
+            if self.graph_registry and self.graph_manager:
```

Comment:
isn't this condition always True after your PR?

Reply:
Yes you are right, probably was left over from an old implementation. I will remove it